src/Fakta.IntegrationTests/Program.fs

```diff
@@ -4,13 +4,29 @@ open Fuchu
 
 [<EntryPoint>]
 let main argv = 
-  Tests.defaultMainThisAssembly argv
-  //Tests.run Agent.tests
-  //Tests.run ACL.tests
-  //Tests.run Catalog.tests
-  //Tests.run Event.tests
-  //Tests.run Health.tests
-  //Tests.run KV.tests
-  //Tests.run Session.tests
-  //Tests.run Status.tests
+  //Tests.defaultMainThisAssembly argv
+  
+  // I don't know how else to make tests to run in the exact order
```

Comment:
Fair enough. What tests depend on what?

You could do

``` fsharp

let tests = [

  ]
  (tests |>List.fold (fun s t -> s + Tests.run) 0)
```

To keep track of exit codes.